engine/runtime/src/test/java/org/enso/compiler/ExecCompilerTest.java

```diff
@@ -68,8 +68,12 @@ public void testHalfAssignment() throws Exception {
 
   @Test
   public void testInvalidEnsoProjectRef() throws Exception {
-    var module = ctx.eval("enso", """
+    var module =
+        ctx.eval(
+            "enso",
+            """
     from Standard.Base.Errors.Common import all
+    from Standard.Base.Meta.Enso_Project import enso_project
```

Comment:
We have to explicitly import `enso_project` here, as `enso_project` is no longer a special generated method on every module. Note that `enso_project` is exported by default with `from Standard.Base import all`.